commit 9031d01af19f900bcf930ec96c2cb891d84c6225 (refs/changes/82/3582/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-10T11:31:02-08:00 (1 year, 7 months ago)
    
    TRITON-226 TRITON-137 broke sapi proto mode setup (i.e. headnode setup) (correctly set config var values)

diff --git a/boot/setup.sh b/boot/setup.sh
index 6222817..6969b9e 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -75,6 +75,7 @@ EOF
     NAPI_URL=http://$(echo "${NAPI_ADMIN_IPS}" | cut -d',' -f1)
     CNAPI_URL=http://$(echo "${CNAPI_ADMIN_IPS}" | cut -d',' -f1)
     VMAPI_URL=http://$(echo "${VMAPI_ADMIN_IPS}" | cut -d',' -f1)
+    SERVER_UUID=$(mdata-get sdc:server_uuid)
 
     # This config file is used during setup to bootstrap SAPI. With the
     # exception that it requires IP addresses instead of DNS names (as binder is
@@ -89,9 +90,9 @@ EOF
     "level": "debug"
   },
   "datacenter_name": "$DATACENTER_NAME",
-  "serviceName": "{{SERVICE_NAME}}",
-  "instanceUuid": "{{auto.ZONENAME}}",
-  "serverUuid": "{{auto.SERVER_UUID}}",
+  "serviceName": "sapi",
+  "instanceUuid": "$ZONE_UUID",
+  "serverUuid": "$SERVER_UUID",
   "adminIp": "$SAPI_ADMIN_IP",
   "moray": {
     "host": "$MORAY_HOST",
